packages/integration-tests/api/api.js

```diff
@@ -394,6 +394,121 @@ async function addProviderApi({ prefix, provider }) {
   });
 }
 
+/**
+ * Fetch a list of providers from the Cumulus API
+ *
+ * @param {Object} params - params
+ * @param {string} params.prefix - the prefix configured for the stack
+ * @returns {Promise<Object>} - the list of providers fetched by the API
+ */
+async function getProviders({ prefix }) {
+  const payload = await callCumulusApi({
+    prefix: prefix,
+    functionName: 'ApiProvidersDefault',
+    payload: {
+      httpMethod: 'GET',
+      resource: '/providers',
+      path: '/providers'
+    }
+  });
+
+  return JSON.parse(payload.body);
```

Comment:
Instead of returning the parsed body, this should just return the response from `callCumulusApi`.  If I wanted to use this function to verify that the API returns some response other than a 200, I wouldn't be able to.  The same goes for `retrieveProvider`.